app/models/user.rb

```diff
@@ -9,6 +9,14 @@ class User < ApplicationRecord
   validates :email, presence: true, uniqueness: true, format: {with: URI::MailTo::EMAIL_REGEXP}
   validates :password, allow_nil: true, length: {minimum: 12}
   validates :password, not_pwned: {message: "might easily be guessed"}
+  validates :lat, numericality: {greater_than_or_equal_to: -90, less_than_or_equal_to: 90}, allow_nil: true
+  validates :lng, numericality: {greater_than_or_equal_to: -180, less_than_or_equal_to: 180}, allow_nil: true
```

Comment:
Do we plan to use a gem like https://github.com/alexreisner/geocoder for this? I think validations, methods to get this populated etc will get added with the questionnaire PRs

Reply:
This'll do for now.